src/marqo/core/exceptions.py

```diff
@@ -102,6 +102,9 @@ def __init__(self, error_message: str,
         self.error_message = error_message
         self.status_code = int(status_code)
 
+    def __str__(self) -> str:
```

Comment:
@vicilliar This change caused test_add_documents_should_fail_a_batch_using_vectorise_per_doc_strategy test to fail due to the exception string representation changes. Could you fix the test? Thanks